front/app/components/SeatInfo/BillingInfo.tsx

```diff
@@ -0,0 +1,172 @@
+  if (isNil(maximumSeatNumber) || isNil(additionalSeats) || !seats) {
+    return null;
+  }
+
+  const usedSeats = {
+    admin: seats.data.attributes.admins_number,
+    collaborator: seats.data.attributes.project_moderators_number,
+  }[seatType];
+  const seatTypeMessage: SeatTypeMessageDescriptor = {
+    admin: messages.adminSeats,
+    collaborator: messages.collaboratorSeats,
+  };
+  const seatTypeTitle = seatTypeMessage[seatType];
+  const seatTypeTooltipMessages: SeatTypeMessageDescriptor = {
+    admin: messages.adminSeatsTooltip,
+    collaborator: messages.collaboratorSeatsTooltip,
+  };
+  const seatTypeTooltipMessage = seatTypeTooltipMessages[seatType];
+  const totalSeats = additionalSeats + maximumSeatNumber;
+  const remainingSeats = totalSeats - usedSeats;
+
+  let totalSeatsBreakdownMessage = formatMessage(messages.seatsWithinPlanText);
+  if (usedSeats > maximumSeatNumber) {
+    totalSeatsBreakdownMessage = formatMessage(messages.seatsExceededPlanText, {
+      noOfSeatsInPlan: maximumSeatNumber,
+      noOfAdditionalSeats: additionalSeats,
+    });
+  }
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      padding="20px"
+      bgColor={rgba(colors.teal400, 0.07)}
+    >
+      <Box display="flex" alignItems="center">
+        {seatType === 'admin' && (
+          <Icon name="shield-checkered" fill={colors.teal300} mr="8px" />
+        )}
+        <Text color="teal700" variant="bodyM" fontWeight="bold" mr="4px">
+          {formatMessage(seatTypeTitle)}
+        </Text>
+        <IconTooltip
+          content={
+            <FormattedMessage
+              {...seatTypeTooltipMessage}
+              values={{
+                visitHelpCenter: (
+                  <a
+                    href={formatMessage(messages.rolesSupportPage)}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    <FormattedMessage {...messages.visitHelpCenter} />
+                  </a>
+                ),
+              }}
+            />
+          }
+        />
+      </Box>
+
+      <Box display="flex">
+        <Box display="flex" flexDirection="column">
+          <Text color="teal700" mr="8px" variant="bodyS" my="0px">
+            {formatMessage(messages.remainingSeats)}
+          </Text>
+          <Text fontSize="xl" mt="4px">
+            {remainingSeats}
+          </Text>
+        </Box>
+        <Box mx="24px">
+          <Text color="teal700" mr="8px" variant="bodyS" my="0px">
+            {formatMessage(messages.usedSeats)}
+          </Text>
+          <Text fontSize="xl" my="4px">
+            {usedSeats}
+          </Text>
+          {!isOnAdminsAndMangersPage && (
+            <StyledLink target="_blank" to={adminsAndMangersLink}>
+              <Text variant="bodyXs" my="0px" color="coolGrey600">
+                {formatMessage(messages.view)}
+              </Text>
+            </StyledLink>
+          )}
```

Comment:
Let's hide this when there are 0 collaborator seats in use?

<img width="117" alt="Screenshot 2023-04-06 at 11 08 15" src="https://user-images.githubusercontent.com/16427929/230330456-b5f310ef-0aae-4974-9d85-2722fbcf1673.png">